src/actions/splitThought.ts

```diff
@@ -57,8 +58,9 @@ const splitThought = (state: State, { path, splitResult }: { path?: Path; splitR
 
     // move children
     state => {
-      // we can safely assume that the cursor has been set to the newly created thought that contains valueRight
       const childNew = getThoughtById(state, head(state.cursor!))
+      if (!childNew) return state
+      // we can safely assume that the cursor has been set to the newly created thought that contains valueRight
```

Comment:
This comment should go above the line with `state.cursor!` since it attempts to explain the `!`.